api/response/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import (Column, String, ForeignKey, Integer, Boolean)
+from helpers.database import Base
+from utilities.utility import Utility
+
+
+class Response(Base, Utility):
+    __tablename__ = 'responses'
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    room_id = Column(Integer, ForeignKey('rooms.id'))
+    question_id = Column(Integer, ForeignKey('questions.id'))
+    rate = Column(Integer, nullable=True)
```

Comment:
As it is, one question can have a text, rate and check answer,
I feel it would have been better if the type of answer depended on the type of question , so questions have a type and the anser would depend on the question.type file, and the logic can be implemented in the mutate method

Reply:
I didn't work on this file, I only rebased from @migot01 branch for creating feedback question

Reply:
I will have a sync with him to rectify this

Reply:
@shemogumbe I have implemented the requested changes